projects/components/src/common/subscription/subscription-tracker.mixin.ts

```diff
@@ -35,8 +35,7 @@ export function SubscriptionTrackerMixin<TBase extends Type<{}>>(Base: TBase) {
             this.tracker.unsubscribeAll();
         }
 
-        ngOnDestroy(): void {
-            this.unsubscribeAll();
-        }
-    };
+        abstract ngOnDestroy(): void;
```

Comment:
Why was this made abstract by removing the call to this.unsubscribeAll? Wouldn't that make all the subclasses duplicate the call to this.unsubscribeAll?

Reply:
Removed in the most recent version, but the logic to call unsubscribeAll is in the constructor of the SubscriptionTracker